src/System.Console/src/System/Console.cs

```diff
@@ -153,35 +130,11 @@ private static TextWriter CreateOutputWriter(Stream outputStream)
         private static StrongBox<bool> _isStdOutRedirected;
         private static StrongBox<bool> _isStdErrRedirected;
 
-        public static bool IsInputRedirected
-        {
-            get
-            {
-                StrongBox<bool> redirected = Volatile.Read(ref _isStdInRedirected) ??
-                    EnsureInitialized(ref _isStdInRedirected, () => new StrongBox<bool>(ConsolePal.IsInputRedirectedCore()));
-                return redirected.Value;
-            }
-        }
+        public static bool IsInputRedirected => EnsureInitializedWrapper(ref _isStdInRedirected, () => new StrongBox<bool>(ConsolePal.IsInputRedirectedCore())).Value;
```

Comment:
This is a very long line.  Please revert the formatting to be more like what it was. Same below.